features/datalist.yml

```
@@ -0,0 +1,24 @@
+name: <datalist>
+description: The `<datalist>` element TODO
```

This isn't great, but maybe a gesture in the right direction?

```suggestion
description: The `<datalist>` element with `<options>` child elements defines a set of recommended values for an `<input>` element, connected by a `list` attribute set to the ID of the `<datalist>`.
```